### PR TITLE
Alias and Index backtick fix

### PR DESCRIPTION
**Pull request by paulstn**
### Description
Fixes non-displaying chart while using backticks for query field (i.e. aliases) issue. Also fixes issue with using span upper cased in queries.

### Issues Resolved
https://github.com/opensearch-project/dashboards-observability/issues/557
https://github.com/opensearch-project/dashboards-observability/issues/509
https://github.com/opensearch-project/dashboards-observability/issues/759

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by YANG-DB — APPROVED** [review]

**Comment by pjfitzgibbons** [comment]
@paulstn If "Issues https://github.com/opensearch-project/dashboards-observability/issues/557 and https://github.com/opensearch-project/dashboards-observability/issues/509 were ultimately" 
Then please update Description accordingly for simpler understanding.

File: public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_fields.tsx [hunk 1]
```diff
@@ -123,9 +122,8 @@ export const DataConfigPanelFields = ({
                   onClick={() => handleServiceEdit(index, sectionName, false)}
                   data-test-subj="viz-config-add-btn"
                 >
-                  {removeBacktick(
-                    obj[CUSTOM_LABEL] || `${isAggregation ? obj.aggregation : ''} ${obj.label}`
-                  )}
+                  {console.log(obj)}
```
Comment (derek-ho) [inline public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_config_panel_fields.tsx:125]: Please remove

File: public/services/data_fetchers/ppl/ppl_data_fetcher.ts [hunk 2]
```diff
@@ -67,6 +67,7 @@ export class PPLDataFetcher extends DataFetcherBase implements IDataFetcher {
 
     if (isEmpty(query)) return;
 
+    console.log(query);
```
Comment (derek-ho) [inline public/services/data_fetchers/ppl/ppl_data_fetcher.ts:70]: please remove this

File: public/services/data_fetchers/ppl/ppl_data_fetcher.ts [hunk 3]
```diff
@@ -101,6 +102,7 @@ export class PPLDataFetcher extends DataFetcherBase implements IDataFetcher {
       this.query[PATTERN_REGEX],
       this.query[FILTERED_PATTERN]
     );
+    console.log(finalQuery);
```
Comment (derek-ho) [inline public/services/data_fetchers/ppl/ppl_data_fetcher.ts:105]: please remove this

File: public/components/visualizations/charts/shared/common.ts [hunk 4]
```diff
@@ -103,23 +107,28 @@ export const preprocessJsonData = (
...
+          value: entry[serieKey],
           x: concatedXaxisLabel,
           breakdown: concatedBreakdownLabel,
           aggName: serieKey,
         };
+        console.log(tabularVizData);
       }
       seriesFlattenedEntries.push(tabularVizData);
     });
+    console.log('----');
```
Comment (derek-ho) [inline public/components/visualizations/charts/shared/common.ts:131]: please remove this

File: public/components/visualizations/charts/shared/common.ts [hunk 5]
```diff
@@ -85,10 +85,14 @@ export const preprocessJsonData = (
   { dimensions, series, breakdowns, span }: Partial<ConfigList>
 ): IIntermediateMapping[] => {
   const seriesFlattenedEntries: IIntermediateMapping[] = [];
+  console.log(dimensions);
+  console.log(series);
+  console.log(span);
```
Comment (derek-ho) [inline public/components/visualizations/charts/shared/common.ts:90]: please remove this

File: public/components/visualizations/charts/shared/common.ts [hunk 6]
```diff
@@ -64,11 +64,11 @@ export const transformPreprocessedDataToTraces = (
...
-export const removeBackTick = (entry: any) => {
-  return {
-    ...mapKeys(entry, (val: any, key: string) => removeBacktick(key)),
-  };
-};
+// export const removeBackTick = (entry: any) => {
+//   return {
+//     ...mapKeys(entry, (val: any, key: string) => removeBacktick(key)),
+//   };
+// };
```
Comment (derek-ho) [inline public/components/visualizations/charts/shared/common.ts:71]: can delete this instead of commenting it out if unused

**Review by derek-ho — COMMENTED** [review]
Can you add a test/screenshot of what this fixes exactly? Hard to verify/understand this change from an outside perspective

**Pull request closed without merging**